src/core_init_atmosphere/mpas_init_atm_gwd.F

```diff
@@ -305,43 +322,221 @@ function compute_gwd_fields(domain) result(iErr)
          end if
       end do
 
-
-      deallocate(topo)
-      deallocate(landuse)
       deallocate(hlanduse)
 
-      iErr = 0
+      iErr = free_tile_list(tilesHead)
 
    end function compute_gwd_fields
 
 
    !***********************************************************************
    !
-   !  function read_global_30s_topo
+   !  subroutine get_box_size_from_lat_lon
```

Comment:
It's a minor issue, but the name of this routine has changed to `get_box_size_from_lat`.